### DISCUSSION TITLE
Binary release for Windows platform remains unmaintained

### DISCUSSION THREAD

**Discussion by yaner-here** [KDE/ghostwriter#889] [General]
I learned that Ghostwriter is still maintained actively, but I can't find a binary release for latest version on Windows platform.

- [Github Official Repo](https://github.com/KDE/ghostwriter/releases): last update for v2.1.6 in Sep 14 2022
- [Chocolate Package-manager source](https://community.chocolatey.org/packages/ghostwriter): last update for v1.5.0 in 17 Feb 2018
- [KDE bianry factory](https://github.com/KDE/ghostwriter/blob/5480dd43c6a7963a7889caa25946b9345130b6c7/README.md?plain=1#L17): has been already shutdown
- [Gitlab Official Repo](https://invent.kde.org/office/ghostwriter/-/releases): no release page provided.

Does any maintained Windows release source exist? Please let me know in case I missed something. If not, will this be in your development roadmap?

**Comment by PrSunflower** [comment]
Hello, its September 2026 and I have just scoured the web in search for a Windows binary... to no avail.

I don't know what's happening to this project, if the Linux version is still actively maintained, ... but it seems like the last Windows version that we have is from 2022 🙁🙁🙁

**Reply by PrSunflower** [reply]
I just noticed the original author of Ghostwriter giving important information in #894 . 👈
